src/button-group/__tests__/icon-button-item.test.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { ButtonGroupProps } from '../../../lib/components/button-group/interfaces';
+import IconButtonItem from '../../../lib/components/button-group/icon-button-item';
```

Comment:
It would be great to test all items types: button, group, and menu. It is better to test the all via the root component, for example:

```
test('sample', () => {
  const wrapper = renderButtonGroup({ items });
  const button = wrapper.findButtonById('test-button');
  // test button
})
```